jvb/src/main/kotlin/org/jitsi/videobridge/cc/config/BitrateControllerConfig.kt

```diff
@@ -25,17 +25,18 @@ class BitrateControllerConfig {
         /**
          * The bandwidth estimation threshold.
          *
-         * In order to limit the resolution changes due to bandwidth changes we
-         * react to bandwidth changes greater bweChangeThresholdPct / 100 of the
-         * last bandwidth estimation.
+         * In order to limit the resolution changes due to bandwidth changes we only react to bandwidth changes greater
+         * than {@code bweChangeThreshold * last_bandwidth_estimation}.
          */
-        private val bweChangeThresholdPct: Int by config {
+        private val bweChangeThreshold: Double by config {
             "org.jitsi.videobridge.BWE_CHANGE_THRESHOLD_PCT".from(JitsiConfig.legacyConfig)
+                .convertFrom<Int> { it / 100.0 }
             "videobridge.cc.bwe-change-threshold-pct".from(JitsiConfig.newConfig)
+                .convertFrom<Int> { it / 100.0 }
```

Comment:
Would it work to read the percentage as a double? It seems like it'd be a backward-compatible change to allow people to set their threshold to 2.5% or whatever.

Reply:
I added a new property without the `-pct` (keeping the existing for backward compatibility). Unfortunately it looks like typesafe doesn't parse `5%` as a number:
`videobridge.cc.bwe-change-threshold-pct has type STRING rather than NUMBER`

Reply:
Yeah, I meant that they could literally type "2.5" (without the % sign).

Reply:
Now they can do either
`videobridge.cc.bwe-change-threshold-pct=2.5` or
`videobridge.cc.bwe-change-threshold=0.025`

Reply:
Can they?  The code is still doing `convertFrom<Int>`

Reply:
Ah, I missed your point. Fixed now.